fimsauthz-api/fims-authz.yaml

```diff
@@ -291,7 +320,23 @@ components:
             For this server, this is satisfied by the use of a UUID.
         scope:
           description: See securityDefinitions section in this spec.
+          type: array
+          items:
+            type: string
+            example: "example.scope.name.read"
+          maxItems: 1
```

Comment:
ASTM wants to move this to no longer be max size=1.
As per https://github.com/astm-utm/Protocol/issues/22 
Recommend changing max to say 10